aten/src/ATen/native/Blas.cpp

```diff
@@ -133,8 +133,19 @@ inline void dot_check(const Tensor& self, const Tensor& other) {
 }
 
 Tensor dot(const Tensor &self, const Tensor &other){
-  at::NoNamesGuard guard;
+  if (self.is_complex()) {
+    if (self.is_conj()) {
+      if (other.is_conj()) {
+        return (at::dot(self.conj(), other.conj())).conj();
```

Comment:
Since `self.is_conj()` here,  would `self.conj()` incur in a copy? Would it not be better to use `_set_conj(bool)` here to avoid this copy? Same for all the other `conj()`. If this does not make a copy, feel free to mark as resolved.

Reply:
that would modify the input tensor

Reply:
Oh, I thought that it was not possible to do `self.conj()` without copying, but it's just that a new tensor is created, but no memory is allocated! Now I understand, thanks.